lib/x509.js

```diff
@@ -2931,7 +2931,13 @@ pki.certificateError = {
  * @param caStore a certificate store to verify against.
  * @param chain the certificate chain to verify, with the root or highest
  *          authority at the end (an array of certificates).
- * @param verify called for every certificate in the chain.
+ * @param options a callback to be called for every certificate in the chain or 
+ *                  an object with:
+ *                  verifyCallback a callback to be called for every certificate 
```

Comment:
Can you rename this to just "verify"?  I don't think we need the "Callback" suffix.

Reply:
Done